File: 백준/Bronze/32722. Juta teekond/Juta teekond.cc
#include <algorithm>
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    int n;
    bool flag = true;
    vector<vector<int>> arr = {{1, 3}, {6, 8}, {2, 5}};
    for (int i = 0; i < 3; i++) {
        cin >> n;
        if (find(arr[i].begin(), arr[i].end(), n) == arr[i].end()) {
            flag = false;
            break;
        }
    }
    cout << (flag ? "JAH" : "EI");
    return 0;
}

File: 백준/Bronze/32722. Juta teekond/Juta teekond.py
import sys
input=lambda:sys.stdin.readline().rstrip()
road=[(1,3),(6,8),(2,5)]
arr=[int(input()) for _ in range(3)]
flag=True
for i in range(3):
    if arr[i] not in road[i]:
        flag=False
        break
print("JAH" if flag else "EI")